src/site/includes/related-benefit-hubs.drupal.liquid

```diff
@@ -12,8 +12,12 @@
           <!-- Show the link if it hasn't been rendered yet -->
           {% unless supportServiceIDs contains supportService.entity.entityId %}
             <li>
-              <p class="vads-u-margin-top--0"><strong>{{ supportService.entity.title }}</strong> <a href="{{ supportService.entity.fieldLink.url.path }}"
-                  rel="noreferrer noopener">{{ supportService.entity.fieldPhoneNumber }}</a></p>
+              <p class="vads-u-margin-top--0"><strong>{{ supportService.entity.title }}</strong>
+              {% if supportService.entity.fieldLink %}
+                <a href="{{ supportService.entity.fieldLink.url.path }}"
+                    rel="noreferrer noopener">{{ supportService.entity.fieldPhoneNumber }}</a>
+              {% endif %}
```

Comment:
This if-statement fixes an Axe error where the 711 message was rendering an empty <a> element.

![image](https://user-images.githubusercontent.com/1915775/97044169-afb71180-1541-11eb-90b8-392341df1bf5.png)